docs/framework/wpf/advanced/toc.yml

```diff
@@ -481,6 +481,8 @@
   - name: Migration and Interoperability
     href: migration-and-interoperability.md
     items:
+    - name: Migrating from WPF 3.X to WPF 4.5
```

Comment:
```suggestion
    - name: Migrate from WPF 3.X to WPF 4.5
```

Reply:
Do you also need to remove the article from the desktop-wpf TOC?

Reply:
oh yeah, duh! ha!